site/en/docs/webstore/i18n/index.md

```diff
@@ -157,134 +149,149 @@ app that has English as its default language and British English as its secondar
 }
 ```
 
-If you add a locale later, you just need to add a `_locales/_locale_/messages.json` file for that
+If you add a locale later, you just need to add a `_locales/LOCALE_CODE/messages.json` file for that
```

Comment:
I think we should tweak this intro a little to clarify the scope of what "you just need to…" applies to. I'd suggest something like this:

If you add a locale later, you just need to add a `_locales/LOCALE_CODE/messages.json` file for that new locale **in order for Chrome Web Store to pick up the localized `name` and `description`**.